### DISCUSSION TITLE
Customize sources in a multi-site group environment

### DISCUSSION THREAD

**Discussion by jan-dh** [craftcms/cms#15750] [Ideas] [answered]
I love the functionality to create Customize sources however, it feels that the feature is missing some extra functionality to really make it work in a multi-site/multi-site group configuration. We're running a website with 1 site group with 8 different countries. Currently setting up a second site group with another 3 countries. Content-wise they are all related and authors from site group A can also edit content of site group B; However, fleshing out the CMS & creating some author-friendly structure is quiet hard since custom sources from site group A always appear on site group B even if some of the channels/sections don't exist in the other site group. Since site groups are a thing in Craft CMS and propagation methods are in place to handle the use case of site groups it would be nice if we can also bring a good author experience for site group by customizing sources on a per site group level?

**Comment by brandonkelly** [comment, marked as answer]
Custom sources have a “Sites” setting (added in Craft 4.5 via #13368). So you can choose which sites each source is shown for.

More granular than by site group, but it should get the job done for you.

**Reply by jan-dh** [reply]
Missed that releasen nice!

**Reply by jan-dh** [reply]
@brandonkelly I'm on 4.12 and can't seem to find the site options on existing sources, only when creating new one or am I missing something?

**Reply by brandonkelly** [reply]
It’s only available for custom sources. Native sources created for sections already only show up for sites the section is actually enabled on.
